ndcube/utils/wcs.py

```diff
@@ -300,3 +158,270 @@ def _pixel_keep(wcs_object):
     if hasattr(wcs_object, "_pixel_keep"):
         return wcs_object._pixel_keep
     return np.arange(wcs_object.pixel_n_dim)
+
+
+def reflect_axis_index(axis, naxes):
+    """Reflects axis index about center of number of axes.
```

Comment:
I find this description confusing. Maybe rename it to something other than reflect?

Reply:
Fair point.  Unless you have a preferred suggestion, I'll rename it ```convert_between_array_and_pixel_axes```.